front-dispatcher/src/strings/strings.ts

```diff
@@ -212,4 +213,5 @@ export const apiUrlsStrings = {
 };
 export const usefulStrings = {
   whiteSpace: "\xa0",
+  Div: "DIV",
```

Comment:
also here for what you need this string ?

Reply:
to differ between the users click target location and not cause a click to add it to history and search input